doc/changes/makedocreldata.g

```diff
@@ -9,7 +9,8 @@ GAPInfo.ManualDataChanges:= rec(
   bookname:= "changes",
   pathtoroot:= "../..",
 
-  files:= [ ],
+  files:= [
```

Comment:
Actually, I only did it to reduce the diffs to the other `makedocreldata.g` files. Other than that, i think we can and should mostly get rid of these files...